lib/airbrake-ruby.rb

```diff
@@ -275,20 +275,19 @@ def create_deploy(deploy_params, notifier = :default)
     private
 
     ##
-    # Calls +method+ on +notifier+ with provided +args+.
+    # Calls +method+ on +notifier+ with provided +args+. If +notifier+ is not
+    # configured, returns an error hash.
     #
-    # @raise [Airbrake::Error] if none of the notifiers exist
+    # @return [Hash{String=>String}, Object]
     def call_notifier(notifier, method, *args, &block)
-      if @notifiers.key?(notifier)
-        @notifiers[notifier].__send__(method, *args, &block)
-      else
-        # If we raise SystemExit, the Ruby process can gracefully quit without
-        # the unwanted Airbrake::Error.
-        raise args.first if args.first.class == SystemExit
-
-        raise Airbrake::Error,
-              "the '#{notifier}' notifier isn't configured"
+      unless @notifiers.key?(notifier)
+        return {
+          'error' => "#{LOG_LABEL} the '#{notifier}' notifier isn't configured, " \
+                     "'#{method}' was not called"
+        }
```

Comment:
1. We still need to return early in order to prevent execution (which will crash), so I assume you want to remove the hash
2. I can see your point and I partially agree with it, but my question is what harm will this hash introduce? If nobody uses the return value, it's still useful. If a customer complains that Airbrake doesn't send errors, my first question will be "what is the return value?". Think of it as a support tool.